addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -172,7 +174,7 @@ def _get_mail_channel(self, anonymous_name, previous_operator_id=None, user_id=N
             'anonymous_name': False if user_id else anonymous_name,
             'country_id': country_id,
             'channel_type': 'livechat',
-            'name': ', '.join([self.env['res.users'].browse(user_id).name if user_id else anonymous_name, operator.livechat_username if operator.livechat_username else operator.name]),
+            'name': ', '.join([visitor_user.name if user_id else anonymous_name, operator.livechat_username if operator.livechat_username else operator.name]),
```

Comment:
For consistency, I would write `visitor_user.name if visitor_user` and instantiate `visitor_user` to `False` l.164.

Reply:
Is it possible to add a test for this fix (or just an assert in an existing test, see test_get_mail_channel.py) ?

Reply:
@mge-odoo changes done!
And added test in https://github.com/odoo/odoo/pull/31346 , is it what you're asking for or should I add another one for this?
Thanks :)

Reply:
You can add the almost same one here (except that there won't be any country name).
When this commit will be forward-ported in master, the tests will be merged in one :) 
Thanks

Reply:
I've added the test in this one, let me know your thoughts.
Thanks